.circleci/config.yml

```diff
@@ -48,7 +48,12 @@ jobs:
           keys:
             - go-mod-v1-{{ checksum "go.sum" }}
 
-      - run: git merge origin/master
+      - run: |
+          git config user.name "Circle CI"
+          git config user.email "ci@mesosphere.com"
+          git checkout -b pr
```

Comment:
The first two lines are great 👍 I am just wondering why you did not stick with `git merge origin/master` ?

Reply:
I was still working on this since I couldn't test it without making a pull request, but basically I misunderstood the error and thought the issue was that we were merging master into the PR instead of the PR into master. That may be an issue (I'm not sure), but the actual issue that I was hitting was the Git configuration.

It should be ready for review now!